server/src/services/pejob.service.ts

```diff
@@ -316,3 +316,37 @@ export const getPeJobFromId = async ({ id, caller }: { id: string; caller: strin
     return manageApiError({ error, api_path: "jobV1/job", caller, errorTitle: "getting job by id from PE" })
   }
 }
+/**
+ * @description Retourne un tableau contenant la seule offre Pôle emploi identifiée
+ */
+export const getPeJobFromIdV2 = async ({ id, caller }: { id: string; caller: string | undefined }): Promise<{ job: ILbaItemPeJob } | null> => {
+  try {
+    const job = await getPeJob(id)
+
+    if (!job) {
+      throw Boom.badRequest()
```

Comment:
```suggestion
      throw Boom.notFound()
```
me paraît plus approprié. Quel est le standard qu'on met en place ?

Reply:
Bah après en avoir discuté avec @moroine, le notFound donne une indication à un potentiel hacker. Après ici il s'agit d'offre PE.